development/gitlab-cli/gitlab-cli.SlackBuild

```diff
@@ -26,30 +26,26 @@ cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=gitlab-cli
 DEBNAM=glab
-VERSION=${VERSION:-1.46.1}
+DEBARCH=amd64
+VERSION=${VERSION:-1.48.0}
 BUILD=${BUILD:-1}
 TAG=${TAG:-_SBo}
 PKGTYPE=${PKGTYPE:-tgz}
 
 if [ -z "$ARCH" ]; then
```

Comment:
This doesn't work if someone has ARCH already set in the env.

The previous code which checked for an empty ARCH, then set it. And then a separate if block built the DEBARCH var based on it was correct.

Could you revert that change?